db/migrate/20161108105410_create_products.rb

```diff
@@ -0,0 +1,17 @@
+class CreateProducts < ActiveRecord::Migration[5.0]
+  def change
+    create_table :products do |t|
+    	
+    	t.string :title
+    	t.boolean :taxable
+    	t.string :featured_image
+    	t.boolean :available
```

Comment:
When you have a boolean column, try to avoid the 3 state boolean problem :) https://robots.thoughtbot.com/avoid-the-threestate-boolean-problem

You can change the boolean columns to add null:false and default: false :)